src/components/pages/coach/CoachDashboard/index.jsx

```diff
@@ -24,11 +38,508 @@ import { ReactComponent as MorningImg } from  '../../../../assets/icons/morning.
+            $lastname: String!
+            $phone: String!
+            $zip: String!
+            $country: String!
+        ){
+        alterUser(
+            userId: $id
+            address: $address
+            city: $city
+            email: $email
+            firstName: $firstname
+            lastName: $lastname
+            telephone: $phone
+            postalCode: $zip
+            country: $country
+        ){
+            result
+            msgCode
+            message
+        }
+    }
+`;
+
+const clientNoCache = new ApolloClient({
+    link: new HttpLink({
+        uri: 'https://manage.pharmaziegasse.at/api/graphql'
+    }),
+    cache: new InMemoryCache(),
+    defaultOptions: {
+        query: {
+            fetchPolicy: 'network-only',
+            errorPolicy: 'all',
+        },
+    }
+});
+const clientOnlyCache = new ApolloClient({
+    link: new HttpLink({
+        uri: 'https://manage.pharmaziegasse.at/api/graphql'
+    }),
+    cache: new InMemoryCache(),
+    defaultOptions: {
+        query: {
+            fetchPolicy: 'cache-first',
+            errorPolicy: 'all',
+        },
+    }
+});
+
 class CoachDashboard extends React.Component{
 
+    state = {
+        modal: false
+    }
+
     componentDidMount = () => {
         // Set page title
         document.title = "Your customers";
+        // Refetch users
+        this._loginUser();
+        this._fetchUsersFirstTime();
+
+        // Get custom greeting info
+        this.getGreetingImg();
+        this.getGreetingQuote();
+    }
+
+    toggle = (user) => {
+        if(!this.state.modal){
+            this.setState({
+                modal: true,
+                modalUser: {
+                    name: user.firstName + " " + user.lastName,
+                    uid: user.id
+                }
+            });
+        } else {
+            this.setState({
+                modal: false,
+                modalUser: undefined,
+            })
+        }
+        
+    }
+
+    syncManually = () => {
+        this.setState({
+            syncing: true,
+            userSyncErrors: false,
+        });
+        this._fetchUsersWithoutCache();
+        this._fetchNodeUsers(this.state.jwt);
+    }
+
+    _fetchNodeUsers = (token) => {
+        const config = {
+            headers: {'Authorization': "bearer " + token}
+        };
+
+        axios.get( 
+        'https://charmnode.herokuapp.com/users',
+        config
+        ).then((response) => {
+            if(response.status === 200){
+                let customers = response.data.customers;
+                // Fetch all Charm users and pass shopify users
+                this._fetchAllUsers(customers);
+            } else if(response.status === 401 || response.status === 403){
+                // Forbidden (token invalid)
+                this._loginUser();
+            } else {
+                this.setState({
+                    syncing: false
+                }, () => console.log("Error",response));
```

Comment:
# please remove `console` (no-console)

In JavaScript that is designed to be executed in the browser, it's considered a best practice to avoid using methods on `console`. Such messages are considered to be for debugging purposes and therefore not suitable to ship to the client. In general, calls using `console` should be stripped before being pushed to production.

```js
console.log("Made it here.");
console.error("That shouldn't have happened.");
```

Examples of **incorrect** code for this rule:

```js
/* eslint no-console: "error" */

console.log("Log a debug level message.");
console.warn("Log a warn level message.");
console.error("Log an error level message.");
```

Reply:
Noted.
However, the console functions do not exist without reason. It is a really great, fully integrated development tool.